experimental/dds/tree2/src/test/snapshots/files/complete-3x3.json

```diff
@@ -27,7 +27,7 @@
value\":\"21\"}],\"FieldB\":[{\"type\":\"TestLeaf\",\"value\":\"22\"},{\"type\":\"TestLeaf\",\"value\":\"23\"},{\"type\":\"TestLeaf\",\"value\":\"24\"}],\"FieldC\":[{\"type\":\"TestLeaf\",\"value\":\"25\"},{\"type\":\"TestLeaf\",\"value\":\"26\"},{\"type\":\"TestLeaf\",\"value\":\"27\"}]}}]],[\"FieldB\",[{\"type\":\"TestInner\",\"fields\":{\"FieldA\":[{\"type\":\"TestLeaf\",\"value\":\"28\"},{\"type\":\"TestLeaf\",\"value\":\"29\"},{\"type\":\"TestLeaf\",\"value\":\"30\"}],\"FieldB\":[{\"type\":\"TestLeaf\",\"value\":\"31\"},{\"type\":\"TestLeaf\",\"value\":\"32\"},{\"type\":\"TestLeaf\",\"value\":\"33\"}],\"FieldC\":[{\"type\":\"TestLeaf\",\"value\":\"34\"},{\"type\":\"TestLeaf\",\"value\":\"35\"},{\"type\":\"TestLeaf\",\"value\":\"36\"}]}},{\"type\":\"TestInner\",\"fields\":{\"FieldA\":[{\"type\":\"TestLeaf\",\"value\":\"37\"},{\"type\":\"TestLeaf\",\"value\":\"38\"},{\"type\":\"TestLeaf\",\"value\":\"39\"}],\"FieldB\":[{\"type\":\"TestLeaf\",\"value\":\"40\"},{\"type\":\"TestLeaf\",\"value\":\"41\"},{\"type\":\"TestLeaf\",\"value\":\"42\"}],\"FieldC\":[{\"type\":\"TestLeaf\",\"value\":\"43\"},{\"type\":\"TestLeaf\",\"value\":\"44\"},{\"type\":\"TestLeaf\",\"value\":\"45\"}]}},{\"type\":\"TestInner\",\"fields\":{\"FieldA\":[{\"type\":\"TestLeaf\",\"value\":\"46\"},{\"type\":\"TestLeaf\",\"value\":\"47\"},{\"type\":\"TestLeaf\",\"value\":\"48\"}],\"FieldB\":[{\"type\":\"TestLeaf\",\"value\":\"49\"},{\"type\":\"TestLeaf\",\"value\":\"50\"},{\"type\":\"TestLeaf\",\"value\":\"51\"}],\"FieldC\":[{\"type\":\"TestLeaf\",\"value\":\"52\"},{\"type\":\"TestLeaf\",\"value\":\"53\"},{\"type\":\"TestLeaf\",\"value\":\"54\"}]}}]],[\"FieldC\",[{\"type\":\"TestInner\",\"fields\":{\"FieldA\":[{\"type\":\"TestLeaf\",\"value\":\"55\"},{\"type\":\"TestLeaf\",\"value\":\"56\"},{\"type\":\"TestLeaf\",\"value\":\"57\"}],\"FieldB\":[{\"type\":\"TestLeaf\",\"value\":\"58\"},{\"type\":\"TestLeaf\",\"value\":\"59\"},{\"type\":\"TestLeaf\",\"value\":\"60\"}],\"FieldC\":[{\"type\":\"TestLeaf\",\"value\":\"61\"},{\"type\":\"TestLeaf\",\"value\":\"62\"},{\"type\":\"TestLeaf\",\"value\":\"63\"}]}},{\"type\":\"TestInner\",\"fields\":{\"FieldA\":[{\"type\":\"TestLeaf\",\"value\":\"64\"},{\"type\":\"TestLeaf\",\"value\":\"65\"},{\"type\":\"TestLeaf\",\"value\":\"66\"}],\"FieldB\":[{\"type\":\"TestLeaf\",\"value\":\"67\"},{\"type\":\"TestLeaf\",\"value\":\"68\"},{\"type\":\"TestLeaf\",\"value\":\"69\"}],\"FieldC\":[{\"type\":\"TestLeaf\",\"value\":\"70\"},{\"type\":\"TestLeaf\",\"value\":\"71\"},{\"type\":\"TestLeaf\",\"value\":\"72\"}]}},{\"type\":\"TestInner\",\"fields\":{\"FieldA\":[{\"type\":\"TestLeaf\",\"value\":\"73\"},{\"type\":\"TestLeaf\",\"value\":\"74\"},{\"type\":\"TestLeaf\",\"value\":\"75\"}],\"FieldB\":[{\"type\":\"TestLeaf\",\"value\":\"76\"},{\"type\":\"TestLeaf\",\"value\":\"77\"},{\"type\":\"TestLeaf\",\"value\":\"78\"}],\"FieldC\":[{\"type\":\"TestLeaf\",\"value\":\"79\"},{\"type\":\"TestLeaf\",\"value\":\"80\"},{\"type\":\"TestLeaf\",\"value\":\"81\"}]}}]]]"
```

Comment:
What's up with all of this? Are these all things that have been detached?

Reply:
These are just fields that were generated detached. I think they weren't showing up before cause they weren't placed under the root and that was prob just wrong.

Reply:
@connorskees could you look into fixing this one as part of the snapshot work? probably makes more sense to attach in this scenario and have a different scenario which checks summarization of detached roots (or maybe summarize at both points in the same scenario--point is what we're checking against snapshots right now is pretty strange and probably an oversight)

Reply:
I added a pointer to this comment to ADO#5326.